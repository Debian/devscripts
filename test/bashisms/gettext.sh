#!/bin/sh

echo $"hello world -- BASHISM"

echo "foo ' bar moo'$"
