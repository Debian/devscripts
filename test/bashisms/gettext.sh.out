possible bashism in bashisms/gettext.sh line 3 ($"foo" should be eval_gettext "foo"):
echo $"hello world -- BASHISM"
